components/admin_console/admin_sidebar_section.jsx

```diff
@@ -61,6 +63,14 @@ export default class AdminSidebarSection extends React.PureComponent {
         if (this.props.subsection) {
             className += ' sidebar-subsection';
         }
+        let tag = '';
+        if (this.props.tag) {
+            tag = (
+                <span className={`${className}-tag`}>
+                    <span>{this.props.tag}</span>
```

Comment:
If we want to pass a string from the parent component instead of a formattedmessage component, then we should be using a formattedMessage here.